server/gobackcomp/go113_pred.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 The NATS Authors
```

Comment:
You have 2 files doing the same. Remove one and maybe you can simply name it go112.go, so we know that when we no longer want to support go112.go, we can remove.

Reply:
yes, just realized my commit did not capture renaming it.
current name is go1.13 prev as it applies to previous versions

Reply:
As we discussed, I think that we could have the "emulation" code in server/errors.go (and not have this new package) and add a TODO that we can then replace with native code when we no longer support Go 1.12. The issue for supporting older release is that the Go client tests build from source and there in Travis we still have Go 1.11, 1.12 and 1.13.